pages/common/arduino-builder.md

```diff
@@ -8,9 +8,9 @@
 
 `arduino-builder -compile {{path/to/sketch.ino}}`
 
-- Specify the debug level (1 to 10, defaults to 5):
+- Specify the debug level (default: 5):
 
-`arduino-builder -debug-level {{level}}`
+`arduino-builder -debug-level {{1,...,10}}`
```

Comment:
```suggestion
`arduino-builder -debug-level {{1,2,3,4,5,6,7,8,9,10}}`
```
How about this? 
Also, I agree with @blueskyson about the command description change.

Reply:
I think we should not present more than `n` numbers in placeholders literally. If there are more than `n` numbers then we can use `from..to` syntax or `from,...,to`. But the point is it should be used everywhere. And rules have to be written to fix this fact.